src/core/rules.js

```diff
@@ -506,6 +566,57 @@ class Rules extends Rule {
         return this;
     }
 
+    shouldExcludeOn(type, feature) {
+        if (this.excludeOn === null) {
+            return false;
+        }
+
+        for (let entry of this.__excludeOn) {
+            if (entry.type !== '*' && entry.type !== type) {
```

Comment:
are wildcards like mytype_* possible which have to be checked here as well?

Reply:
Since they are resolved using the regex earlier (see comment above), every valid combination is checked.

Reply:
I replaced the given magic strings by already defined constants